docs/platforms/android/index.mdx

```diff
@@ -74,10 +74,12 @@ The wizard will prompt you to log in to Sentry. It'll then automatically do the
 Configuration is done via the application `AndroidManifest.xml`. Here's an example config which should get you started:
 
 
-```xml {filename:AndroidManifest.xml} {"onboardingOptions": {"performance": "4-5", "profiling": "6-7"}}
+```xml {filename:AndroidManifest.xml} {"onboardingOptions": {"performance": "6-7", "profiling": "8-9"}}
 <application>
   <!-- Required: set your sentry.io project identifier (DSN) -->
   <meta-data android:name="io.sentry.dsn" android:value="___PUBLIC_DSN___" />
+  <!-- Add data like request headers and device name, see https://docs.sentry.io/platforms/android/data-management/data-collected/ for more info -->
```

Comment:
`m` I think we should also add this to the [manual setup page](https://docs.sentry.io/platforms/android/manual-setup/)

Reply:
Updated, see below